src/main/webapp/js/community-page-loader.js

```diff
@@ -18,12 +20,18 @@
        * Builds a list element that contains a link to a user page, e.g.
        * <li><a href="/user-page.html?user=test@example.com">test@example.com</a></li>
        */
-      function buildUserListItem(user) {
+      function buildUserListItem(user, aboutMe) {
```

Comment:
Please add to the function documentation comment describing what user and aboutMe (preferably with examples!)